dpnp/tests/third_party/cupy/linalg_tests/test_solve.py

```diff
@@ -60,10 +60,9 @@ def test_solve(self):
         # for other cases this signature must be followed
         # (..., m, m), (..., m, n) -> (..., m, n)
         # https://github.com/numpy/numpy/pull/25914
-        if numpy.lib.NumpyVersion(numpy.__version__) < "2.0.0":
-            self.check_x((2, 4, 4), (2, 4))
-            self.check_x((2, 3, 2, 2), (2, 3, 2))
-            self.check_x((0, 2, 2), (0, 2))
+        if numpy.lib.NumpyVersion(numpy.__version__) >= "2.0.0":
```

Comment:
I believe we can align that with the latest version of CuPy tests, where the whole test is marked with `@testing.with_requires("numpy>=2.0")`.